dashboard/src/index.css

```diff
@@ -16,3 +15,9 @@ svg.icon {
   position: relative;
   bottom: -0.125em;
 }
+
+/* Override Hex underline text-decoration in buttons in alerts
+see: https://github.com/bitnami/hex/issues/216 */
+.alert .button {
+  text-decoration: none;
+}
```

Comment:
why don't you add here the same code to remove the `underlying` decoration for the links?

```
a,
a:hover {
  text-decoration: none !important;
}
```

Reply:
We should avoid this kind of rules because the underlying is important for readability

Reply:
@andresmgot @Angelmmiguel yes my intention was only to remove the underlines in buttons, I think if Hex wants us to underline regular links, we should follow that.